director/template/template.go

```diff
@@ -172,25 +173,43 @@ func (l varsLookup) Get(name string) (interface{}, bool, error) {
 	}
 
 	if len(splitName) > 1 {
-		tokens := []patch.Token{patch.RootToken{}}
-
-		for _, token := range splitName[1:] {
-			tokens = append(tokens, patch.KeyToken{Key: token})
-		}
-
-		findOp := patch.FindOp{
-			Path: patch.NewPointer(tokens),
+		pointer, err := patch.NewPointerFromString("/" + strings.Join(splitName[1:], "/"))
+		if err != nil {
+			return nil, false, err
 		}
 
-		val, err = findOp.Apply(val)
+		val, found, err = processPatchPointer(pointer, val)
```

Comment:
Is found used after this assignment? It seems redundant if it's only set when err != nil.

Reply:
Maybe? Golang refactoring did things for me, I'll double check.